src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -284,6 +284,8 @@ private enum TimeStampType
 
         public static double ConvertTimeIntervalToHeight(DateTime start, DateTime end, int scaleMode)
         {
+            if (end <= start) return 0;

```

Comment:
as discussed in Slack, this line doesn't fix the given bug, and having `end < start` potentially uncovers other issues, so it might be better to address those separately

Reply:
OK, agree with you. Already removed this from fix.